client/src/app/components/pages/static/FaqPage.tsx

```diff
@@ -55,9 +55,10 @@ export default function FaqPage({}: Props) {
           <a href="#multiple-files">send one file at a time</a>.
         </li>
         <li>
-          File size limits: In this initial stage, it is only possible to send
-          files of up to 200 MB. When sending files to mobile devices, this
-          number may be smaller than 100 MB.
+          File size limits: In this initial stage, because of the browser APIs,
+          the receiver needs to hold the entire contents of the file in memory.
+	  Therefore, it is only possible to send files of up to 200 MB. When
```

Comment:
I think this isn't "really" the reason but just one contributing factor. Basically someone decided to limit at 200mb because that actually worked, and for unknown reasons biggers sizes worked worse. I didn't do this testing, so don't know the details.

I believe another reason was "since it's free, and all data traverses our server, maybe a limit like this is good / fine"...?

Reply:
Thank you @meejah and @wuan for these inputs.
I've reworded the whole section.
@meejah: do you think this last version is giving a more accurate explanation?